Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -138,14 +142,23 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
                 }
             });
 
-            this.set('version', new Y.eZ.Version());
-            this.set('languageCode',e.selectedLanguageCode);
-            view.set('languageCode', e.selectedLanguageCode);
+            Y.Object.each(fields, function (field) {
```

Comment:
I have provided method `_setFields` which is setting fields on given content and version and then it sets updated content and version attributes of view service. I haven't refactored it more as fields are bit differrent depending on context - when initializing models fields are populated with default values, on the other hand when changing the language fields are populated with data received in EventFacade.